app/components/gender-select.hbs

```diff
@@ -0,0 +1,16 @@
+<fieldset>
+  <legend class="au-u-hidden">Geslacht</legend>
+  <ul>
+    {{#each this.loadGendersTask.lastSuccessful.value as |gender|}}
```

Comment:
This looks a bit weird at the moment:

![image](https://user-images.githubusercontent.com/3533236/131324860-0f3900cf-1e18-43a1-bab5-3750b851a422.png)

The alignTop argument doesn't seem to work anymore after some of the previous styling changes, but this can be improved in a different PR.

Reply:
Now that I think of it, if we're adding more than 3 options it probably should be a select instead. That would bypass this issue.

Reply:
I would go for a select field.

Reply:
Done